brave/brave-core
# fix(wallet): ENS Offchain Lookup Review Order

## Description 
- Fixes ENS Offchain Lookup review order bug.
- Removes unneeded redux state for send hook.


Resolves <https://github.com/brave/brave-browser/issues/27661>

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. Go to brave://settings/web3 and make sure `Allow ENS offchain lookup` is set to `Ask`
2. Go to brave://wallet/send, choose a token on the `Ethereum` network, enter an amount to send and paste in `offchainexample.eth`
3. You should see the `ENS Offchain` message
4. Click on the `Use ENS domain` button
5. The `Message` should disappear and resolve the address.
6. You should then be able to click `Review Order` and approve the transaction.

https://user-images.githubusercontent.com/40611140/210902276-aa7611f7-4969-4051-9083-ecdbd24ca985.mov
